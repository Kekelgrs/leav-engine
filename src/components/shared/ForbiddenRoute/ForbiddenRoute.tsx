import React from 'react';
import {WithNamespaces, withNamespaces} from 'react-i18next';
import {Icon} from 'semantic-ui-react';

function ForbiddenRoute({t}: WithNamespaces): JSX.Element {
    return (
        <h3>
            <Icon name="ban" />
            {t('admin.forbidden_route')}
        </h3>
    );
}

export default withNamespaces()(ForbiddenRoute);
