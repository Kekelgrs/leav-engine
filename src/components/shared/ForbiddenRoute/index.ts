import ForbiddenRoute from './ForbiddenRoute';
export default ForbiddenRoute;
