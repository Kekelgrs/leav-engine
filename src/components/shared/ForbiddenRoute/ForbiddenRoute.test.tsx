import React from 'react';
import {render} from 'enzyme';
import ForbiddenRoute from './ForbiddenRoute';

describe('ForbiddenRoute', () => {
    test('Snapshot test', async () => {
        const comp = render(<ForbiddenRoute />);

        expect(comp).toMatchSnapshot();
    });
});
