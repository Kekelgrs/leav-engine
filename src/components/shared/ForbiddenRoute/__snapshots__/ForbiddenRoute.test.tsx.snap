// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForbiddenRoute Snapshot test 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "aria-hidden": "true",
          "class": "ban icon",
        },
        "children": Array [],
        "name": "i",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "data": "admin.forbidden_route",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "text",
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "aria-hidden": undefined,
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "aria-hidden": undefined,
          "class": undefined,
        },
      },
      Object {
        "data": "admin.forbidden_route",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "aria-hidden": "true",
            "class": "ban icon",
          },
          "children": Array [],
          "name": "i",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "aria-hidden": undefined,
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "aria-hidden": undefined,
            "class": undefined,
          },
        },
        "type": "text",
      },
    ],
    "name": "h3",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
